include/flatbuffers/idl.h

```diff
@@ -342,14 +341,17 @@ inline bool EqualByName(const Type &a, const Type &b) {
          (a.enum_def == b.enum_def || a.enum_def->name == b.enum_def->name);
 }
 
-struct RPCCall {
+struct RPCCall : public Definition {
+  Offset<reflection::RPCCall> Serialize(FlatBufferBuilder *builder, const Parser &parser) const;
+
   std::string name;
```

Comment:
=> attributes removed (note: the use of `doc_comment` from the base class instead of the dedicated and now removed `rpc_comment` triggered small changes elsewhere)